migration/article/db/migrate/20210422081610_create_articles.rb

```diff
@@ -1,6 +1,6 @@
 class CreateArticles < ActiveRecord::Migration[6.0]
   def change
-    create_table :articles do |t|
+    create_table :articles, id: :serial do |t|
```

Comment:
We must remember to include `id: :serial` in the db migrations to create the table with a type of `BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE`, where bigint unsigned is an 64-bit uint, equivalent to the 64-bit uint type in go.

Reply:
:nice